### ISSUE TITLE
Combine mode

### ISSUE THREAD

**Issue by morzey19** [Stephan-S/FS19_AutoDrive#66]
Is the unload combine mode only for grain harvester? tried the beet and potato harvester and it dosent seem to pickup. great mod by the way.

**Comment by Stephan-S** [comment]
Hey, yes for now it is limited to combines only. I will try to include beet and potato harvesters when I have the time for it.

**Comment by morzey19** [comment]
Ok no problem thanks for your reply. and does the combine unload work with courseplay or Ai vehicle extension?

**Comment by Stephan-S** [comment]
I haven't tried the AI vehicle extension yet, but I don't see why not.
Courseplay however, not yet. Since the driver is only called when the pipe is out (because then the pipe location is known and doesn't have to be calculated), it doesn't work with courseplay. Since the pipe is only extended when a trailer is close by as far as i know.
But I am working on a fix for that too.

**Comment by Stephan-S** [comment]
Hey Roby, I would like to analyze the 'too heavy' aspect.
In SP, I can see ~2% CPU usage per AD driver on my old i5.
Also when it is in unload combine mode.
I also checked in MP and analyzed the network traffic without seeing any spikes when using the unloader.
How many drivers were active when you had the issue? So an estimation of all AI helpers (AD + Giants) would be nice.

**Comment by Roby1164** [comment]
We are 3 on a dedi server, I7 7700K, 32Gb, from VeryGames.
Last night, when the problem occurred, we had 4 or 5 AI's in operation, some with standard AI's and some with AI vehicle extension including my combine, in AI vehicle extension and the trailer truck that had the AD running, and was waiting for the combine, everyone is 3 we had trouble shooting, while if I turned off your AD, everything went back to normal. In SP, even if they have the CP, AI vehicle extension, GPS, this problem has not been tried with two Combines in AI Vehicle, and 2 tractors with trailers with active AD waiting. In this case, however, I had the problem that, after unloading, both the combine harvesters and the trailers, after being unloaded, got stuck, I had to turn off the AI ​​Vehcle on the combines, move them forward, to restart them, and hand-hold the trailers to unload them, they didn't move by themselves, I tried to put them on the path, towards the silos, but, they left, they passed by the silo without stopping and they returned to the field, waiting for the combine, even if they were full.
